core/model/src/test/java/org/eclipse/rdf4j/model/util/RDFCollectionsTest.java

```diff
@@ -71,35 +70,34 @@ public void testConversionRoundtrip() {
 
 	}
 
-	@Test
-	public void testNonWellformedCollection() {
+	@Test(expected = ModelException.class)
+	public void testNonWellformedCollection_MissingTerminator() {
```

Comment:
Not sure why this shows up in your PR, as these changes are already present on the master branch. Possibly caused by your fork being out of sync with the main repo. Puzzling, but not a major problem, it will solve itself when we merge.

Reply:
Apart from this, by the way, I think the fix now looks really good, thanks!

Reply:
Ok sounds good ! Thank you for your collaboration and I fixed this it was my fault ..